ui/propertyeditor/propertyeditorfactory.cpp

```diff
@@ -45,6 +45,7 @@ PropertyEditorFactory::PropertyEditorFactory()
     initBuiltInTypes();
 
     addEditor(QVariant::Color, new QStandardItemEditorCreator<PropertyColorEditor>());
+    addEditor(QVariant::ByteArray, new QStandardItemEditorCreator<PropertyTextEditor>());
```

Comment:
Hm what if the QString is not ascii encoded ? does it keep the correct chars like éàç ?